style.css

```diff
@@ -0,0 +1,77 @@
+body {
+  display: flex;
+  justify-content: center;
+}
+
+.wrapper {
+  width: 375px;
+}
+
+.nav-bar {
+  width: 100%;
+  height: 48px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.nav-bar p {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+}
+
+.header {
+  width: 100%;
+  min-height: 587px;
+  background: rgba(255, 255, 255, 0.925) url("img/bg.png") no-repeat;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  flex-wrap: wrap;
+}
```

Comment:
- [ ] Consider adding margin right and left for `24px` to match like the Figma template.
- [ ] Also kindly remove the height for social media icons to not be stretched
- [ ] [OPTIONAL] It will be better to export icons as `SVG` because it saves the quality of the icons.

| Your design | Figma Design |
| :---: | :---: |
| ![image](https://user-images.githubusercontent.com/53402600/170369986-e3d8e157-e878-42c7-8177-4fedbaeaafca.png) | ![image](https://user-images.githubusercontent.com/53402600/170369902-1cfd9ad7-df32-4463-a993-f6f80f5a668f.png)|